azurerm/internal/services/automation/resource_arm_automation_schedule.go

```diff
@@ -82,7 +82,7 @@ func resourceArmAutomationSchedule() *schema.Resource {
 				Optional:         true,
 				Computed:         true,
 				DiffSuppressFunc: suppress.RFC3339Time,
-				ValidateFunc:     validate.RFC3339DateInFutureBy(time.Duration(5) * time.Minute),
+				ValidateFunc:     validate.RFC3339Time,
```

Comment:
Since we are removing this constraint, could we add the constraint/check into the create/update function?

Reply:
Thanks for the suggestion. I have added the check into create/update function.